app/assets/javascripts/school_administrator_dashboard/dashboard_components/discipline_dashboard/SchoolDisciplineDashboard.js

```diff
@@ -21,17 +21,11 @@ class SchoolDisciplineDashboard extends React.Component {
       selectedChart: 'location',
       selectedCategory: null
     };
-    this.setDate = this.setDate.bind(this);
     this.setStudentList = this.setStudentList.bind(this);
     this.resetStudentList = this.resetStudentList.bind(this);
     this.selectChart = this.selectChart.bind(this);
   }
 
-  setDate(range) {
```

Comment:
This is removed leftover code? 👍

Reply:
Yup. Just something leftover from the date slider, but doesn't have anything to do with this PR in particular.